README.md

```diff
@@ -8,8 +8,8 @@ Publish HASS events to your [Elasticsearch](https://elastic.co) cluster!
 ## Features
 
 - Efficiently publishes Home-Assistant events to Elasticsearch using the Bulk API
-- Automatically maintains Indexes and Index Templates using Index Lifecycle Management ("ILM")
-- Supports [X-Pack Security](https://www.elastic.co/products/x-pack/security) via optional username and password
+- Automatically sets up Datastreams or maintains Indexes and Index Templates using Index Lifecycle Management ("ILM") depending on your cluster's capabilities
+- Supports [X-Pack Security](https://www.elastic.co/products/x-pack/security) via optional username, password, and API keys

```

Comment:
Since we're editing this, what do you think about removing the rather historic "X-Pack" naming?

```suggestion
- Supports Elatsic's [stack security features](https://www.elastic.co/elastic-stack/security) via optional username, password, and API keys

```